packages/ui/src/composables/useForm.ts

```diff
@@ -3,13 +3,15 @@ import { computed, PropType } from 'vue'
 export interface UseFormProps {
   disabled: boolean;
   readonly: boolean;
+  required?: boolean;
   id?: string | number;
   name?: string | number;
 }
 
 export const useFormProps = {
   disabled: { type: Boolean, default: false },
   readonly: { type: Boolean, default: false },
+  required: { type: Boolean, default: false },
```

Comment:
We need `show-required-mark` prop. For example, we have a form where all inputs required.

Reply:
From my point of view we need this mark to just highlight for the user that this field is required.